### DISCUSSION TITLE
Getting different readings on LilyGO devices

### DISCUSSION THREAD

**Discussion by leifa71** [cyberman54/ESP32-Paxcounter#952] [General]
Hi,

We are a couple of guys trying to figure how this works, we have tried to adjust the RSSI Limit so it only counts devices "nearby" so we can monitor for example a room. We have 3 different devices with the same code and on 1 counter we get a reading of 26 while on the other just 1... so what could be the case here. We also tried walking in a crowded place and the first time it counted 60 mac addresses (wich was more or less validated), a moment later we did the same procedure and only ended up with a count of 8. 

Setting: WiFi on, BLE off
             RSSI Limit= -40
             
So do anyone know have any ideas around this ?

This i sreally a cool project and i would like to make it work so that the devices are more in-synch with the counting so i can trust the values.

usecase: Counting "people" for example in a auditorium, store, building, restaurant etc. but for this only power the device on for 5 minutes, walk around with it and then check the reading. The  later on do the same walking around after a new power on. So it's for checking different areas at a specific momnent.

Best regards

Leif Andersson

**Comment by cyberman54** [comment]
rssi -40 is a far to tight value. Use values  between -70 and -100.

Don't expect absolute precise counting results, i.e. not when number of people are small. Sniffing isca heuristic. It works good with bigger crowds, and when you compare relative values over time. You can measure the flow of pax, not the absolute number.

**Reply by leifa71** [reply]
I have tried from  not filtered and down to -30 and still get very various readings. Yesterday i tried alot of different ways to get the mac adress to log to the serial port aswell to investigate wich device i actually count, but probably i was to stupid for that :) Maybe more tries tonight. But what i saw, was that when i turn my wifi on my phone off, i had 0 on the counter (onboard display) for a long time and when i activated WiFi it got instantly 6-8 directly... and this value was exactly as the amount of WiFi access points that showed up in my Iphone. any ideas ?
